Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM gcr.io/distroless/static
+FROM gcr.io/distroless/static:nonroot
+
+USER 65532:65532
```

Comment:
We should not add the USER id. Also, for further information check GoogleContainerTools/distroless#235 
However, we need to ensure that the image will be distroless and rootless.

Reply:
I made a PR to solve this issue see: https://github.com/brancz/kube-rbac-proxy/pull/86/files